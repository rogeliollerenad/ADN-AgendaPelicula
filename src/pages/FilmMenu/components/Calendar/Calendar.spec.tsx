import { render } from '@testing-library/react';
import React from 'react';
import MockDate from 'mockdate';
import { MaterialUIPickers } from './index';

describe('Prueba Calendario', () => {
  const setSelectedDate = jest.fn();
  const selectedDate = new Date('2010-12-17T03:24:00');

  const props = {
    setSelectedDate,
    selectedDate,
  };

  it('should match snapshot', () => {
    const { container } = render(<MaterialUIPickers {...props} />);
    expect(container).toMatchSnapshot();
  });
});
