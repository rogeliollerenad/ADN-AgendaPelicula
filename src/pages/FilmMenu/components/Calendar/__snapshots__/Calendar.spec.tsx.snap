// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba Calendario should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container"
    style="justify-content: space-around;"
  >
    <div
      class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      data-cy="select-day"
      data-testid="day_picker"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
        data-shrink="true"
        for="date-picker-inline"
        id="date-picker-inline-label"
      >
        Selecciona Fecha
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
          id="date-picker-inline"
          type="text"
          value="12/17/2010"
        />
        <div
          class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
        >
          <button
            aria-label="change date"
            class="MuiButtonBase-root MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                />
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
      
    </div>
    <div
      class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      data-cy="select-hour"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
        data-shrink="true"
        for="time-picker"
        id="time-picker-label"
      >
        Selecciona Hora
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
          id="time-picker"
          type="text"
          value="03:24 AM"
        />
        <div
          class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
        >
          <button
            aria-label="change time"
            class="MuiButtonBase-root MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                />
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;
